### ISSUE TITLE
Bug when closing the app and than opening it from the multitasking bar

### ISSUE THREAD

**Issue by vmirchev** [bvogelzang/BVReorderTableView#22]
While I was reordering a friend called me, which closed the application. Afterwards I opened the same application from the multitasking bar and the draggingView was not closed and the reordering was not finished and there is no way to finish it.

I solved it by adding a applicationDidBecomeActiveNotification which removed the draggingView and replaced the dummy string with the saved object, by calling the delegate method with indexPath param - currentLocationIndexPath.

If i find time I will clear my code and commit the fix, if it has not already been fixed in the newest stable version.

**Comment by mluisbrown** [comment]
Good catch! Just reproduced the bug in my own app which uses BVReorderTableView.

**Comment by NickEntin** [comment]
Was able to reproduce it as well in my app.  Would love to see your fix.
